lib/ansible/module_utils/facts/system/service_mgr.py

```diff
@@ -52,6 +52,17 @@ def is_systemd_managed(module):
                     return True
         return False
 
+    @staticmethod
+    def is_systemd_managed_offline(module):
+        # tools must be installed
+        if module.get_bin_path('systemctl'):
+            # check if /sbin/init is a symlink to systemd, or does not exist
+            if not os.path.exists('/sbin/init'):
+                return True
```

Comment:
@bcoca if I were to remove this section, would you be okay merging it?  This would leave only the SUSE family of distros out in the cold.  All other systemd-based distros I tried have `/sbin/init` as a symlink to `systemd`.  Same for debian jessie when in systemd mode.  If you switch it to sysVinit mode, `/sbin/init` is no longer a symlink to `systemd` but is instead the SysV binary.

I could find no non-systemd distro where `systemctl` is an available package and `/sbin/init` is missing...  If you can identify such a distro, I can look to see how to handle it.

FWIW, the linux service_mgr code has been unmodified since you originally introduced it, except for a small tweak for openrc.

Reply:
off top of my head, gentoo (i used it w/o systemd, but with systemctl installed), but mostly i would just look at previous tickets that posed the issue (i installed systemctl on my workstation just to test this).

Reply:
Thanks for checking.  I've removed these lines.  I've found that OpenSUSE and friends have an optional `systemd-sysvinit` package that provides this symlink.  (Their initrd has `/init` symlinked to `/usr/lib/systemd/systemd`, so no technical need for /sbin/init to be symlinked given that they're using an initrd.)